addons/mail/static/tests/tours/mail_full_composer_test_tour.js

```diff
@@ -0,0 +1,73 @@
+odoo.define('mail/static/tests/tours/mail_full_composer_test_tour.js', function (require) {
+"use strict";
+
+const tour = require('web_tour.tour');
+
+/**
+ * This tour depends on data created by python test in charge of launching it.
+ * It is not intended to work when launched from interface. It is needed to test
+ * an action (action manager) which is not possible to test with QUnit.
+ * @see mail/tests/test_mail_full_composer.py
+ */
+tour.register('mail/static/tests/tours/mail_full_composer_test_tour.js', {
+    test: true,
+}, [{
+    content: "Click on Send Message",
+    trigger: '.o_ChatterTopbar_buttonSendMessage',
+},{
+    content: "Write something in composer",
+    trigger: '.o_ComposerTextInput_textarea',
+    run: 'text blahblah',
+},{
+    content: "Open full composer",
+    trigger: '.o_Composer_buttonFullComposer',
+},{
+    content: "Check subject is autofilled",
+    trigger: 'input[name="subject"]',
+    run: () => {
+        const subjectValue = document.querySelector('input[name="subject"]').value;
+        if (subjectValue !== "Re: Test User") {
+            tour._consume_tour(
+                tour.running_tour,
+                `Full composer should have "Re: Test User" in subject input (actual : ${subjectValue})`
+            );
```

Comment:
I think all this does is to` console.error` the message.

Reply:
It makes the tour fail and print that message, yes

Reply:
I mean just use console.error instead of calling an unrelated private method just to do console error.